code/_core/obj/item/clothing/pants/goblin_loin.dm

```diff
@@ -0,0 +1,36 @@
+/obj/item/clothing/pants/goblin_loin
+	name = "Goblin King's loin"
+	desc = "You monster, stealing from the little green men."
+	desc_extended = "The loin is imbued with the life essence of the late Goblin King. Greatly desired by mages."
+	icon = 'icons/obj/item/clothing/pants/goblin_loin.dmi'
+
+	defense_rating = list(
+		ARCANE = AP_GREATSWORD,
+		HOLY = -AP_GREATSWORD,
+		DARK = AP_GREATSWORD
+	)
+
+	rarity = RARITY_LEGENDARY
+
+	size = SIZE_2
+
+	value = 1000
+
+	var/cooldown_effect = 0
+
+/obj/item/clothing/pants/goblin_loin/Finalize()
+	. = ..()
+	start_thinking(src)
+
+/obj/item/clothing/pants/goblin_loin/think()
```

Comment:
I think it would be better if the ``start_thinking(src)`` was only called when the loincloth is worn. I'm imagining a scenario where someone has near infinite mana regen by putting 10 pairs of goblin loins in their bluespace backpack.

Reply:
yea, I was planning on changing that.